packages/transmute-framework/src/stream-model/__tests__/sync.spec.js

```diff
@@ -100,4 +100,28 @@ describe('sync', () => {
     expect(streamModel2.state.lastIndex).toBe(1);
     // console.log(JSON.stringify(streamModel2.state, null, 2));
   });
+
+  it('calling sync repeatedly returns the same state', async () => {
+    let newEventStore = await eventStore.clone(accounts[0]);
```

Comment:
Why not use the `eventStore` variable from `beforeAll` ?

Reply:
I agree, these tests are very old... I want to clean them up.